packages/config/src/api/site_info.ts

```diff
@@ -117,13 +154,20 @@ const getIntegrations = async function ({
 
   const baseUrl = new URL(host ? `http://${host}` : `https://api.netlifysdk.com`)
 
+  const useV2Endpoint = featureFlags?.cli_integration_installations_meta
+
+  const url = useV2Endpoint
+    ? `${baseUrl}team/${accountId}/integrations/installations/meta`
+    : `${baseUrl}site/${siteId}/integrations/safe`
+
   try {
-    const response = await fetch(`${baseUrl}site/${siteId}/integrations/safe`)
+    const response = await fetch(url)
 
     const integrations = await response.json()
     return Array.isArray(integrations) ? integrations : []
   } catch (error) {
-    // for now, we'll just ignore errors, as this is early days
+    // Integrations should not block the build if they fail to load
+    // TODO: We should consider blocking the build as integrations are a critical part of the build process
```

Comment:
NIT: @lemusthelroy have you created a linear issue for it if yes maybe link it in the comment. (So this TODO does not get forgotten)

Reply:
I will create an issue for this and add it to the comment 👍

Reply:
Done